packages/blade/src/utils/useColorScheme/useColorScheme.ts

```diff
@@ -0,0 +1,31 @@
+import { useState, useCallback } from 'react';
+import { ColorSchemeNames, colorSchemeNames } from '../../tokens/theme';
+import getColorScheme from '../getColorScheme';
+
+export type UseColorScheme = {
+  colorScheme: Exclude<ColorSchemeNames, 'system'>;
```

Comment:
No it becomes confusing. Because then I have to maintain and keep all the instances of those types in sync

Reply:
You will have to maintain it only in 1 place. Right now you'll have to take the efforts of changing it everywhere in case you need to or make mistakes like misspelling 'system'.